docker-compose.yml

```diff
@@ -36,12 +42,68 @@ services:
             - AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION=False
         ports:
             - "8081:8080"
-        extra_hosts:
-            # just if you have a local CKAN instance running in _nginx_ host in you local machine
-            # export HOST_IP=`ip -4 addr show scope global dev docker0 | grep inet | awk '{print \$2}' | cut -d / -f 1`
-            - "nginx:${HOST_IP}"
-        healthcheck:
-            test: ["CMD-SHELL", "[ -f /usr/local/airflow/airflow-webserver.pid ]"]
-            interval: 30s
-            timeout: 30s
-            retries: 3
+
+    ckan:
+        image: viderum/ckan-cloud-docker:ckan-latest-datagov-theme
+        restart: always
+        volumes:
+            - ./docker-extras/docker-compose/ckan-secrets.sh:/etc/ckan-conf/secrets/secrets.sh
+            - ./docker-extras/docker-compose/ckan-conf-templates:/etc/ckan-conf/templates
+            - ckan-data:/var/lib/ckan
+        environment:
+            - CKAN_STORAGE_PATH=/var/lib/ckan/data
+            - CKAN_K8S_SECRETS=/etc/ckan-conf/secrets/secrets.sh
+            - CKAN_K8S_TEMPLATES=/etc/ckan-conf/templates
+            - GUNICORN_WORKERS=2
+            - CKAN_CONFIG_TEMPLATE_PREFIX=datagov-theme-
```

Comment:
can probably change this to something more generic and rename the files accordingly

Reply:
I did [some changes](https://github.com/datopian/ckan-ng-harvest/commit/5357683817c41a719e7e91109a3100d8276c1a27). I think now is easy to understand.